modules/globebrowsing/globebrowsingmodule_lua.inl

```diff
@@ -370,33 +371,81 @@ getLocalPositionFromGeo(std::string globeIdentifier, double latitude, double lon
 
 /**
  * Get geographic coordinates of the camera position in latitude, longitude, and altitude
- * (degrees and meters).
+ * (degrees and meters). If the optional bool paramater is specified, the camera 
+ * eye postion will be used instead
  */
-[[codegen::luawrap]] std::tuple<double, double, double> getGeoPositionForCamera() {
+[[codegen::luawrap]] std::tuple<double, double, double> 
+getGeoPositionForCamera(std::optional<bool> useEyePosition) 
+{
     using namespace openspace;
     using namespace globebrowsing;
 
     GlobeBrowsingModule* module = global::moduleEngine->module<GlobeBrowsingModule>();
-    const RenderableGlobe* globe = module->castFocusNodeRenderableToGlobe();
+    const RenderableGlobe* globe = module->castFocusNodeRenderableToGlobe();//focus vs anchor
     if (!globe) {
         throw ghoul::lua::LuaError("Focus node must be a RenderableGlobe");
     }
+    Camera* camera = global::navigationHandler->camera();
+
+    glm::dvec3 cameraPosition = camera->positionVec3();
+
 
-    const glm::dvec3 cameraPosition = global::navigationHandler->camera()->positionVec3();
     const SceneGraphNode* anchor =
-        global::navigationHandler->orbitalNavigator().anchorNode();
+        global::navigationHandler->orbitalNavigator().anchorNode(); //focus vs anchor
```

Comment:
Should the comment be there?  If yes, I would say add a bit more text and put it on its own line